utils/theme.tsx

```diff
@@ -0,0 +1,20 @@
+import { Source_Sans_3 } from 'next/font/google';
+import { text } from 'stream/consumers';
+
+export const SourceSans = Source_Sans_3({
+  weight: ['400', '600', '700'],
+  subsets: ['latin'],
+  variable: '--font-source-sans',
+});
+
+const prototypeStylingTheme = {
```

Comment:
no doubt Joshua will want to go to town on this...for now I've arbitrarily cherry-picked some colors from the Pro theme.